test/livebook/runtime/k8s_test.exs

```diff
@@ -0,0 +1,141 @@
+defmodule Livebook.Runtime.K8sTest do
+  alias Livebook.Runtime
+  use ExUnit.Case, async: true
+
+  # To run these tests, install [Kind](https://kind.sigs.k8s.io/) on your machine.
+  @moduletag :k8s
+
+  @assert_receive_timeout 10_000
+  @cluster_name "livebook-runtime-test"
+  @kubeconfig_path "tmp/k8s_runtime/kubeconfig.yaml"
+  @default_pod_template """
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    generateName: livebook-runtime-
+    labels:
+      livebook.dev/runtime: integration-test
+  spec:
+    containers:
+      - image: ghcr.io/livebook-dev/livebook:nightly
+        name: livebook-runtime
+        env:
+          - name: TEST_VAR
+            value: present
+
+  """
+
+  setup_all do
+    {clusters_out, exit_code} = System.cmd("kind", ~w(get clusters))
+    assert 0 == exit_code, "kind is not installed. Please install kind."
+
+    if not (clusters_out
+            |> String.split("\n", trim: true)
+            |> Enum.member?(@cluster_name)) do
+      exit_code = Mix.Shell.IO.cmd("kind create cluster --name #{@cluster_name}", quiet: true)
```

Comment:
Let's use `System.cmd` in all cases. We could use `Mix.shell().cmd`, but I don't think it's necessary here.

```suggestion
      {_, exit_code} = System.cmd("kind create cluster --name #{@cluster_name}")
```

Reply:
```
  0) Livebook.Runtime.K8sTest: failure on setup_all callback, all tests have been invalidated
     ** (UndefinedFunctionError) function Mix.Shell.cmd/1 is undefined or private. Did you mean:

           * cmd/2
           * cmd/3
```

Reply:
System.cmd had a totally different API. But I can rewrite it, no big deal... Any reason for this?

Reply:
But also, ethere is no `:quiet` option which makes the test super verbose (prints all stdout from docker build etc)!

Reply:
Do `into: ""`, and it will output into the string, which you can ignore!

Reply:
mmh... according to the docs, `into: ""` is the default. And it still prints to stdout. You mean sth like this, right?

```ex
    System.cmd(
      "docker",
      ~w(buildx build --build-arg BASE_IMAGE=hexpm/elixir:#{bindings[:elixir]}-erlang-#{bindings[:otp]}-ubuntu-#{bindings[:ubuntu]} --build-arg VARIANT=default --load -t ghcr.io/livebook-dev/livebook:nightly .),
      into: ""
    )

```

Reply:
ah, so stderr_to_stdout: true?

Reply:
that just redirects stderr to stdout, no? But let me try

Reply:
yeah, but stdout goes into the string, so we should be good !

Reply:
oh so this verbose output was all stderr?? no way...

Reply:
anyway, it does work. I really don't understand why. But I've pushed the changes.